src/include/reconcile.h

```diff
@@ -396,3 +396,12 @@ typedef struct {
         (r)->ref->page->modify->rec_result == WT_PM_REC_REPLACE ||                              \
         ((r)->ref->page->modify->rec_result == WT_PM_REC_MULTIBLOCK &&                          \
           (r)->ref->page->modify->mod_multi_entries == 1))
+
+/* Called when building the internal page image. */
```

Comment:
This comment is pretty useless as-is, could you please add some explanation of what it does and what the output is?

Reply:
I think it could also be combined with `WT_BUILD_DELTA_LEAF`, since 80% of the checks are identical.